platform/ui/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon, Typography } from '@ohif/ui';
+
+const Dropdown = ({ titleElement, title, list }) => {
+  const [open, setOpen] = useState(false);
+  const element = useRef(null);
+
+  const renderTitleElement = () => {
```

Comment:
titleElement is kinda ambiguous to me. If this is an element socket, where its located? lets document this prop and how to use it.

Reply:
agree! I kept this approach from the old Dropdown component, but I ended up changing a little the component implementation, avoiding ambiguity by removing the title/titleElement. The children itself is the "title" and it can be customizable. A Dropdown Icon `˅` will be visible by default to have a basic dropdown style if the title is a single string. It can be hidden through the new prop `showDropdownIcon`. Let me know your opinion on this new approach